lib/libmbedtls/mbedtls/library/gcm.c

```diff
@@ -90,6 +90,26 @@ void mbedtls_gcm_init( mbedtls_gcm_context *ctx )
     memset( ctx, 0, sizeof( mbedtls_gcm_context ) );
 }
 
+int mbedtls_gcm_clone( mbedtls_gcm_context *dst,
+                       const mbedtls_gcm_context *src )
+{
+    if( dst == NULL || src == NULL )
+    {
+        return( MBEDTLS_ERR_CIPHER_BAD_INPUT_DATA );
+    }
+
+    mbedtls_cipher_clone( &dst->cipher_ctx, &src->cipher_ctx );
+    memcpy( dst->HL, src->HL, 16 * sizeof(uint64_t ) );
+    memcpy( dst->HH, src->HH, 16 * sizeof(uint64_t ) );
```

Comment:
minor: s/`sizeof(uint64_t )`/`sizeof( uint64_t )`/    (here and 1 line below)

Reply:
Yes, I will fix it.